invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/LoanMetadata/LoanMetadata.js

```diff
@@ -19,6 +27,57 @@ export default class LoanMetadata extends Component {
     this.setState({ isAvailableItemsVisible: newState });
   };
 
+  _renderItem() {
+    const data = this.props.loanDetails;
+    return (
+      !_isEmpty(data.metadata.item) && (
+        <ItemMetadata
+          item={data.metadata.item}
+          loanState={data.metadata.state}
+          changeItemClickHandler={this.showAvailableItems}
+        />
+      )
+    );
+  }
+
+  _renderDeletedItem() {
+    const data = this.props.loanDetails;
+    return (
+      _isEmpty(data.metadata.item) &&
+      data.metadata.item_pid && (
+        <Message
+          warning
+          icon="trash alternate"
+          header={`The Item ${
+            data.metadata.item_pid
+          } assigned to this loan has been deleted!`}
+          content={[
+            'If you need further assistance contact ',
+            <a
+              href={`mailto:cds-support@cern.ch?subject=Deleted Item ${
```

Comment:
We cannot hardcode such email :)
You have to expose yet again another config from config.py, `SUPPORT_EMAIL`